Misc/NEWS.d/next/C API/2019-06-11-02-50-38.bpo-37221.4tClQT.rst

```diff
@@ -0,0 +1,5 @@
+The new function :c:func:`PyCode_NewWithPosOnlyArgs` substitutes
+:c:func:`PyCode_New` as the prefered way to create code objects, having an
+extra parameter representing the number of positional only parameters
+(**posonlyargcount**).  The old function is kept for backward compatibility
+reasons and will set the number os positional only parameters to 0.
```

Comment:
I'm not sure that it's needed to deprecate/prefer anything so far. Just mention that you add the function, no?